core/ui/components/drive_card.tscn

```diff
@@ -0,0 +1,86 @@
+[gd_scene load_steps=6 format=3 uid="uid://caeaxm6st4a4u"]
+
+[ext_resource type="Script" path="res://core/ui/components/drive_card.gd" id="1_jlhco"]
+[ext_resource type="PackedScene" uid="uid://uljtdvmuol3l" path="res://core/systems/input/focus_group_setter.tscn" id="2_jm832"]
+[ext_resource type="PackedScene" uid="uid://c71ayw7pcw6u6" path="res://core/ui/components/card_button.tscn" id="3_84mee"]
+[ext_resource type="PackedScene" uid="uid://8m20p2s0v5gb" path="res://core/systems/input/focus_group.tscn" id="3_c72py"]
+[ext_resource type="PackedScene" uid="uid://bkdsn268g6hx7" path="res://core/ui/components/partition_card.tscn" id="4_wgas0"]
+
+[node name="DriveCard" type="MarginContainer"]
+custom_minimum_size = Vector2(340, 200)
+script = ExtResource("1_jlhco")
+
+[node name="BackgroundPanel" type="Panel" parent="."]
+layout_mode = 2
+
+[node name="FocusGroupSetter" parent="." node_paths=PackedStringArray("target") instance=ExtResource("2_jm832")]
+target = NodePath("../EdgeMarginContainer/LayoutHBox/DriveDataVBox/DriveFocusGroup")
+on_signal = "pressed"
+
+[node name="EdgeMarginContainer" type="MarginContainer" parent="."]
+layout_mode = 2
+theme_override_constants/margin_left = 20
+theme_override_constants/margin_top = 20
+theme_override_constants/margin_right = 20
+theme_override_constants/margin_bottom = 20
+
+[node name="LayoutHBox" type="HBoxContainer" parent="EdgeMarginContainer"]
+layout_mode = 2
```

Comment:
Should add `theme_override_constants/separation` so there is a bit of spacing between the icon, drive, and partitions.